waspc/packages/wasp-config/src/mapUserSpecToAppSpecDecls.ts

```diff
@@ -228,9 +212,10 @@ function mapAuth(
   return {
     userEntity: parseEntityRef(userEntity),
     // TODO: Abstract away this pattern
-    ...(externalAuthEntity && {
-      externalAuthEntity: parseEntityRef(externalAuthEntity),
-    }),
+    externalAuthEntity:
```

Comment:
For entities you did `entities: entities && entities.map(parseEntityRef),` why are you are explicitly checking for `undefined`?

Reply:
Because it can technically be an empty string.